src/gui_tool/gui/general_gui.py

```diff
@@ -69,13 +69,12 @@ def start(self):
             self.cleanup()
 
     def set_GUI(self):
-        cv2.namedWindow(self.WINDOW_NAME)
-        cv2.resizeWindow(self.WINDOW_NAME, self.vcm.get_width(),
-                         self.vcm.get_height() + self.IMG_STARTING_Y)
+        cv2.namedWindow(self.WINDOW_NAME, cv2.WINDOW_NORMAL | cv2.WINDOW_GUI_NORMAL)
```

Comment:
Adding the flags `cv2.WINDOW_NORMAL | cv2.WINDOW_GUI_NORMAL` gives the ability to resize the window while correctly rescaling user input.